### ISSUE TITLE
Can't use arbitrary data file for annotation app upload

### ISSUE THREAD

**Issue by olsonanl** [PATRIC3/oxcart#3]
I want to feed the output from my assembly run into annotation, but it is in /olson/olson/output and I can't specify that folder as an input folder for the annotation app.

**Comment by nconrad** [comment]
yeah, sounds very useful.  I plan on removing that input folder thing at the top and using a folder/file selector "widget" when the user needs to select something, similar to this: https://github.com/PATRIC3/Workspace/issues/11 .  With all the features planned for that widget**, the input folder concept at the top will become useless.

**Imagine a folder/file selector that ignores incompatible types, has search, and also has little feeds of recent folders and recent files.  Data you are sharing shall be clearly marked as well (related to https://github.com/PATRIC3/Workspace/issues/6 ).
